src/pfe/portal/modules/ExtensionList.js

```diff
@@ -24,7 +24,7 @@ const extensionsDir = '/extensions';
 const extensionsPattern = /^(\S+)-(\d+\.\d+\.\d+)\.zip$/; // e.g. extension-name-0.0.1.zip
 const suffixOld = '__old';
 const odoExtensionName = "codewind-odo-extension";
-const odoBinarySource = "https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64"
+const odoBinarySource = "https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64"
```

Comment:
Can we specify a version instead of downloading latest?

Reply:
Good point, we can lock the version to use ODO 1.0.0 (which is a good stable version) for our 0.5.0, then let master uses the latest.